decay-tree-fitter.md

```diff
@@ -63,10 +63,14 @@ root -l DVntuple.root
 TupleDstToD0pi_D0ToKpi->cd()
 DecayTree->StartViewer()
 ```
-Plotting the raw mass of the D* (without the fit) `Dstar_MM` you should see a broad signal around 2 GeV:
+Plotting the raw mass of the D* (without the fit) `Dstar_M` you should see a broad signal around 2 GeV:
 
 <img src="./img/DstarRaw.png" alt="Dstar raw" style="width: 500px;"/>
 
+> ## Which mass variable to use {.callout}
+> In many ntuples you also find a mass variable called `_MM`. This confusingly refers to measured mass. However, it is usually better to use `_M`. `_MM` is the sum of the 4-momenta of the final state particles at the fitted vertex position, but not the result of the actual vertex fit.
+
```

Comment:
To be more explicit, maybe “…sum of the 4-momenta of the final state particles extrapolated back to the fitted vertex position“. To get them thinking about the extrapolation itself.